js/components/JournalEntryForm/JournalEntryForm.js

```diff
@@ -0,0 +1,170 @@
+        }}
+        render={({ handleSubmit, pristine, invalid }) => (
+          <View style={styles.formView}>
+            <Field name="content">
+              {({ input, meta }) => (
+                <TextInput
+                  style={styles.contentInput}
+                  placeholder="Write anything on your mind..."
+                  {...input}
+                />
+              )}
+            </Field>
+            <Field name="tags">
+              {({ input, meta }) => (
+                <TextInput
+                  style={styles.tagsInput}
+                  placeholder="Add #emotions (optional)"
+                  {...input}
+                />
+              )}
+            </Field>
+            <Field name="images">
+              {({ input, meta }) => (
+                <View style={styles.cameraIconContainer}>
+                  <TouchableOpacity
+                    onPress={() => {
+                      console.log("Image picker clicked");
+                      this.showImagePicker();
+                    }}
+                  >
+                    <Image
+                      source={require("../../assets/Icons/camera-icon.png")}
+                      style={styles.cameraIcon}
+                    />
+                  </TouchableOpacity>
+                  {this.state.imageSource &&
+                    this.state.imageSource.map((img, index) => {
+                      return (
+                        <View key={index}>
+                          {console.log(img)}
+                          <TouchableOpacity
+                            onPress={() => {
+                              this.removeImg(img);
+                            }}
+                            style={styles.removeButton}
+                          >
+                            <Ionicon
+                              name="ios-remove-circle"
+                              style={styles.removeIcon}
+                            />
+                          </TouchableOpacity>
+                          <Image source={img} style={styles.images} />
+                        </View>
+                      );
+                    })}
+                </View>
+              )}
+            </Field>
+            <LinearGradient
+              start={{ x: 1, y: 0 }}
+              end={{ x: 1, y: 1 }}
+              colors={[
+                globalStyles.blueGradientColor.start.color,
+                globalStyles.blueGradientColor.end.color
+              ]}
+              style={styles.submitButton}
+            >
+              <TouchableOpacity
+                onPress={() => {
+                  handleSubmit();
+                }}
+                style={styles.submitButton}
+              >
+                <Text style={styles.submitText}>submit</Text>
+              </TouchableOpacity>
+            </LinearGradient>
+            <TouchableOpacity
+              onPress={() => {
+                console.log("submit");
```

Comment:
you should be calling your on submit method not console logging a message.

Reply:
I will use this onPress event to navigate to the Journal Screen. When I receive the navigation props from the person who is using my feature, I will then add the functionality. Right now the functionality that works is the uploading the journal to SpeakBox's api